src/main/resources/assets/reignofnether/lang/en_us.json

```diff
@@ -1,42 +1,677 @@
+    "abilities.reignofnether.connect_portal.tooltip1": "Connect to a transport portal you own anywhere else.",
+    "abilities.reignofnether.connect_portal.tooltip2": "Overrides any existing connections on both sides.",
+    "abilities.reignofnether.connect_porta.error1": "Must target another completed transport portal you own.",
+    "abilities.reignofnether.disconnect_portal": "Sever Connection",
+    "abilities.reignofnether.dismount": "Dismount",
+    "abilities.reignofnether.eject": "Eject",
+    "abilities.reignofnether.explode": "Explode",
+    "abilities.reignofnether.fire_wall_shot": "Fire Wall Shot",
+    "abilities.reignofnether.fire_wall_shot.tooltip1": "\uE004  %ds  \uE005  ",
+    "abilities.reignofnether.fire_wall_shot.tooltip2": "Fire an accurate shot that ignites all blocks in a line.",
+    "abilities.reignofnether.fire_wall_shot.tooltip3": "While this is on cooldown, the blaze cannot attack.",
+    "abilities.reignofnether.go_to_portal": "Go to connected portal",
+    "abilities.reignofnether.mount_hoglin": "Mount Hoglin (Right click to auto-find)",
+    "abilities.reignofnether.mount_hoglin.error1": "Invalid target!",
+    "abilities.reignofnether.mount_ravager": "Mount Ravager (Right click to auto-find)",
+    "abilities.reignofnether.mount_ravager.error1": "Invalid target!",
+    "abilities.reignofnether.mount_ravager.tooltip1": "Mounted pillagers gain explosive arrows",
+    "abilities.reignofnether.mount_spider": "Mount Spider (Right click to auto-find)",
+    "abilities.reignofnether.mount_spider.error1": "Invalid target!",
+    "abilities.reignofnether.promote_illager": "Promote Illager",
+    "abilities.reignofnether.promote_illager.error1": "Unit is too far away!",
+    "abilities.reignofnether.promote_illager.error2": "That unit is already a captain!",
+    "abilities.reignofnether.promote_illager.error3": "You don't own that unit!",
+    "abilities.reignofnether.promote_illager.error4": "Only Vindicators, Pillagers and Evokers may be promoted",
+    "abilities.reignofnether.promote_illager.tooltip1": "\uE004  %ds  \uE005  ",
+    "abilities.reignofnether.promote_illager.tooltip2": "Promote an illager to a captain, giving it a banner that gives",
+    "abilities.reignofnether.promote_illager.tooltip3": "a speed buff to all friendly units in a %d block radius.",
+    "abilities.reignofnether.promote_illager.tooltip4": "You may only have one captain at a time per upgraded castle.",
+    "abilities.reignofnether.roar": "Roar",
+    "abilities.reignofnether.roar.tooltip1": "\uE006  %d   \uE004  %ds  \uE005   ",
+    "abilities.reignofnether.roar.tooltip2": "Emit a deafening roar, knocking away, damaging",
+    "abilities.reignofnether.roar.tooltip3": "and briefly slowing down all nearby enemies. ",
+    "abilities.reignofnether.evoker_fangs_circular": "Evoker Fangs (Circular)",
+    "abilities.reignofnether.evoker_fangs_circular.tooltip1": "\\uE006  %d  \\uE004  %ds  \\uE005  ",
```

Comment:
`"abilities.reignofnether.evoker_fangs_circular.tooltip1": "\uE006  %d  \uE004  %ds  \uE005  ",`

Reply:
Thanks! I missed that